docker-compose.yaml

```diff
@@ -62,7 +62,9 @@ services:
       context: ./spark
       dockerfile: Dockerfile.builder
     volumes:
-      - ./spark:/opt/spark/work-dir
+      - ./spark/src:/opt/spark/work-dir/spark-apps
+      - ./.build:/opt/spark/work-dir/build
+      - ./spark/scripts/start-spark-builder.sh:/opt/spark/work-dir/start.sh
```

Comment:
This file seems to be missing from your commit

Reply:
Which one ? I did not define any script for the builder